src/inactivity/window.ts

```diff
@@ -0,0 +1,42 @@
+import { BrowserView, BrowserWindow, IpcMainEvent } from 'electron'
+import { throttle } from 'lodash'
+import { ACTIVITY_EVENT, INACTIVITY_THROTTLE } from './utils'
+
+export async function watchForInactivity(baseWindow: BrowserWindow | BrowserView, timeoutDuration: number, callback: (baseWindow: BrowserWindow | BrowserView) => any) {
+  let timeout: number | undefined = undefined
```

Comment:
```suggestion
  let timeout?: number
```

Reply:
I can't use this syntax, it's not a class property
<img width="662" alt="Screenshot 2020-11-16 at 10 45 40" src="https://user-images.githubusercontent.com/5877409/99237554-ee668180-27f8-11eb-9612-3c6b7cc014cc.png">